packages/vats/src/lib-board.js

```diff
@@ -1,22 +1,55 @@
 // @ts-check
+/**
+ * @file lib-board: share objects by way of plain-data ids
+ *
+ * @see prepareBoardKit()
+ */
 
-import { assert, details as X, q, Fail } from '@agoric/assert';
+import { assert, Fail, q } from '@agoric/assert';
+import {
+  M,
+  makeScalarBigMapStore,
+  prepareExoClassKit,
+  provideDurableMapStore,
+} from '@agoric/vat-data';
 import { E, Far } from '@endo/far';
 import { makeMarshal } from '@endo/marshal';
-import { makeScalarMapStore } from '@agoric/store';
-import { crc6 } from './crc.js';
 
-import './types.js';
+import { crc6 } from './crc.js';
 
 export const DEFAULT_CRC_DIGITS = 2;
 export const DEFAULT_PREFIX = 'board0';
 
+// #region Interface Guards
+// TODO import from Endo
+const CapDataShape = { body: M.string(), slots: M.array() };
+const MarshalI = M.interface('Marshaller', {
+  serialize: M.call(M.any()).returns(CapDataShape),
+  unserialize: M.call(CapDataShape).returns(M.any()),
+});
+
+const IdShape = M.string();
+const ValShape = M.key();
+const BoardI = {
```

Comment:
```suggestion
const BoardKitIKit = {
```
(cf. https://github.com/Agoric/agoric-sdk/blob/efd964db27052459bd55193b5ff82b9f2b8aab1d/packages/notifier/src/publish-kit.js#L121 )